common/locales/en/front.json

```diff
@@ -252,5 +252,6 @@
     "onlySocialAttachLocal": "Local authentication can be added to only a social account.",
     "invalidReqParams": "Invalid request parameters.",
     "memberIdRequired": "\"member\" must be a valid UUID.",
-    "heroIdRequired": "\"heroId\" must be a valid UUID."
+    "heroIdRequired": "\"heroId\" must be a valid UUID.",
+    "cannotFulfillReq":"Your request cannot be fulfilled.  Email admin@habitica.com if this error persists."
```

Comment:
There's a discussion about it in the Aspiring Coders guild, but in brief, since we've configured the forgot password form to not leak information about which email addresses are in use, it seems sensible to do the same as best we can for changing your email address.

Also, if someone is trying to use an email address that already exists, they would probably benefit from admin assistance. Copying from one of my posts in that guild:

The situation would arise if they'd forgotten that they already had another account, possibly created a long time ago (we'd want them to know about it because it would have good loot for them!), or because they were mistyping their own email address and might need someone to point that out to them :), or because someone else was incorrectly using their email address either maliciously or by accident (definitely a situation that admins should investigate!).